CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 6.0.4
+
+- Avoid the dependency from ActionCable, introduced in Rails 5. Related to https://github.com/alphagov/govuk_frontend_toolkit_gem/pull/15
```

Comment:
I think here, you should just mention that there are no changes, and that this release is being done to release changes in the corresponding Gem.

Talking about anything Ruby or Rails related is a bit out of context, and for example, isn't applicable if you are using the Node module, or some other release.